src/components/dialogs/network-modifications/limit-sets/limit-sets-tabular-modification-form.tsx

```diff
@@ -0,0 +1,268 @@
+            clearErrors(MODIFICATIONS_TABLE);
+            setValue(MODIFICATIONS_TABLE, results.data, {
+                shouldDirty: true,
+            });
+            setIsFetching(false);
+            // For shunt compensators, display a warning message if maxSusceptance is modified along with shuntCompensatorType or maxQAtNominalV
+            if (
+                results.data.some(
+                    (modification) =>
+                        modification.maxSusceptance &&
+                        (modification.shuntCompensatorType || modification.maxQAtNominalV)
+                )
+            ) {
+                snackWarning({
+                    messageId: 'TabularModificationShuntWarning',
+                });
+            }
+        },
+        [clearErrors, setValue, snackWarning]
+    );
+
+    const equipmentType = useWatch({
+        name: TYPE,
+    });
+
+    const amountTemporaryLimits = useWatch({
+        name: AMOUNT_TEMPORARY_LIMITS,
+    });
+
+    const computeRepeatableColumns = useCallback(() => {
+        const columns: RepeatableColumn[] = [];
+        trigger(AMOUNT_TEMPORARY_LIMITS).then((valid) => {
+            if (valid) {
+                for (let i = 1; i <= amountTemporaryLimits; i++) {
+                    LIMIT_SETS_TABULAR_MODIFICATION_REPEATABLE_FIELDS.forEach((field) =>
+                        columns.push({
+                            id: field + i,
+                            name: field,
+                            index: i,
+                        })
+                    );
+                }
+                setReapeatableColumns(columns);
+            }
+        });
+    }, [amountTemporaryLimits, trigger]);
+
+    const csvColumns = useMemo<RepeatableColumn[]>(() => {
+        return [...LIMIT_SETS_TABULAR_MODIFICATION_FIXED_FIELDS.map((id) => ({ id: id })), ...repeatableColumns];
+    }, [repeatableColumns]);
+
+    const csvTranslatedColumns = useMemo(() => {
+        return LIMIT_SETS_TABULAR_MODIFICATION_FIXED_FIELDS.map((field) => {
+            return intl.formatMessage({ id: field });
+        });
+    }, [intl]);
+
+    const commentLines = useMemo(() => {
+        return generateLimitSetCommentLines({
+            csvTranslatedColumns,
+            intl,
+            language,
+        });
+    }, [intl, csvTranslatedColumns, language]);
+
+    const [typeChangedTrigger, setTypeChangedTrigger] = useState(false);
+    const [selectedFile, FileField, selectedFileError] = useCSVPicker({
+        label: 'ImportModifications',
+        header: csvColumns.map((column) => column.id),
+        disabled: !csvColumns,
+        resetTrigger: typeChangedTrigger,
+        language: language,
+    });
+
+    const watchTable = useWatch({
+        name: MODIFICATIONS_TABLE,
+    });
+
+    useEffect(() => {
+        computeRepeatableColumns();
+    }, [computeRepeatableColumns]);
+
+    useEffect(() => {
+        setIsFetching(dataFetching);
+    }, [dataFetching]);
+
```

Comment:
I think we must check, on import, if the different fields in the csv are of the right type, and possibly of the expected value

side should be ONE or TWO
permanentLimit should be a numeric
modificationType should be ADDED or MODIFIED or REPLACED
temporaryLimitsModificationType should ADDED or MODIFIED or DELETED or REPLACED
acceptableDuration should be a numeric
value should be a numeric

For the enum type, the skeleton should also contain these expected values

This is to be consistent with the checks that have recently been added on the other tabular creation/modification